lego/apps/comments/models.py

```diff
@@ -27,5 +27,14 @@ class Meta:
         ordering = ("created_at",)
         permission_handler = CommentPermissionHandler()
 
+    def delete(self):
+        with transaction.atomic():
+            self.created_by = None
+            self.text = "<p>Slettet</p>"
```

Comment:
Does it make more sense to set this to `null`, and handle it in the frontend? Kinda hard to change the behavior if we save this string in the db.

(I have no idea if our `ContentField` will handle it properly without modification tho.)